package.json

```diff
@@ -2,25 +2,42 @@
     "private": true,
     "scripts": {
         "dev": "npm run development",
-        "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "development": "node_modules/cross-env/src/bin/cross-env.js NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
```

Comment:
Why did this need to change?

Reply:
This will makes it much easier when running on our local environments, where normally cross-env and other node modules are not installed into the bin by default. This is why we always get errors for `cross-env not found`, and then have to `yarn global add cross-env` on our environments